ja_jp/assets

```diff
@@ -0,0 +1 @@
+/Users/Henner/Source/puppet-quest-guide/assets
```

Comment:
Oh yeah, good catch. Not entirely sure how git/GitHub handles symlinks, but I guess these should be relative rather than absolute?